tests/test_iot/test_iot.py

```diff
@@ -463,6 +463,37 @@ def test_list_things_with_attribute_and_thing_type_filter_and_next_token():
     )
 
 
+@mock_iot
+def test_endpoints():
+    client = boto3.client("iot", region_name="ap-northeast-1")
+
+    # iot:Data
+    endpoint = client.describe_endpoint(endpointType="iot:Data")
+    endpoint.should.have.key("endpointAddress").which.should_not.contain("ats")
+    endpoint.should.have.key("endpointAddress").which.should.contain("iot")
```

Comment:
I think it makes sense to make these checks more strict, i.e. `should.contain("iot.ap-northeast-1.amazonaws.com")`

Reply:
sounds good.